seedlet/assets/sass/abstracts/_config.scss

```diff
@@ -65,4 +65,7 @@ $typescale-ratio: 1.2; // Run ratio math on 1em == $typescale-base * $typescale-
 	--global--border-radius-lg: #{$typescale-root};
 	--global--border-radius-pill: #{10 * $typescale-root};
 	--global--border-radius-none: 0;
+
+	/* WordPress Variables */
+	--wp--style--color--link: var(--global--color-primary);
```

Comment:
Sorry my previous comment wasn't clear.  By changing to `var( --wp--style--color--link, var(--global--color-primary) );` in the actual link styles, I believe this mapping becomes unnecessary.
```suggestion
```

Reply:
Ah, you're right. That does make sense for now.